packages/api/src/moderation/const/labels.ts

```diff
@@ -99,23 +99,23 @@ export const LABELS: LabelDefinitionMap = {
     strings: {
       settings: {
         en: {
-          name: 'Requested Hidden to Logged-out Users',
+          name: 'Sign-in Required',
```

Comment:
guess this codegen just didn't quite make it into a previous PR?

wanna make sure this PR contains the correct content & not main